src/App/Console.php

```diff
@@ -35,12 +34,8 @@ public function __construct(Application $console, Kernel $app)
         $this->console->add($app->get('console.generate_database'));
         // Set up logger.
         $this->logger = $app->get('logger');
-        try {
-            $this->console->add($app->get('console.populate_rules'));
-            $this->console->add($app->get('console.queue'));
-        } catch (SqsException $e) {
-            $this->logger->debug('Cannot connect to SQS so some commands are not available', ['exception' => $e]);
-        }
+        $this->console->add($app->get('console.populate_rules'));
+        $this->console->add($app->get('console.queue'));
```

Comment:
This will break the `queue:create` command since it doesn't get past these lines without a queue, can work around locally though